### PR TITLE
Protonvpn ng update 2.2.0 to 2.2.2

### PR DESCRIPTION
**Pull request by JeffLabonte**
###### Motivation for this change

I use this VPN and I was getting messages telling me to upgrade. I wanted to give this to the community

###### Things done

Update version from 2.2.0 to 2.2.2, update the sha256 sum, and add myself as a maintainer of this package.

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by jtcoolen** [comment]
Built and tested locally, it works fine. Could you please merge your two commits into one?

**Comment by JeffLabonte** [comment]
> Built and tested locally, it works fine. Could you please merge your two commits into one?

Is there a reason you would prefer to have it into one commit? is it for the backport? I can take care of that.

**Comment by jtcoolen** [comment]
The second change is really part of the first one, so one commit suffices.

**Comment by JeffLabonte** [comment]
@jtcoolen done!

**Review by jtcoolen — APPROVED** [review]
Perfect! Thanks for contributing! Could you merge this change @worldofpeace?

**Comment by JeffLabonte** [comment]
@jtcoolen I will take care of backporting this to 19.09 and 20.03 later today! Thank you for your contributions as well my dear fellow Nix User :)

File: pkgs/applications/networking/protonvpn-cli-ng/default.nix [hunk 1]
```diff
@@ -27,7 +27,7 @@ python3Packages.buildPythonApplication rec {
   meta = with stdenv.lib; {
```
Comment (bhipple) [inline pkgs/applications/networking/protonvpn-cli-ng/default.nix]:
```suggestion
  meta = with lib; {
```
And then delete the `stdenv` parameter entirely from the top-level function call arguments (right now you have both lib and the stdenv, but you only need lib).

File: pkgs/applications/networking/protonvpn-cli-ng/default.nix [hunk 2]
```diff
@@ -2,13 +2,13 @@
 
 python3Packages.buildPythonApplication rec {
   name = "protonvpn-cli-ng";
```
Comment (bhipple) [inline pkgs/applications/networking/protonvpn-cli-ng/default.nix]:
```suggestion
  pname = "protonvpn-cli-ng";
```

File: pkgs/applications/networking/protonvpn-cli-ng/default.nix [hunk 3]
```diff
@@ -27,7 +27,7 @@ python3Packages.buildPythonApplication rec {
   meta = with stdenv.lib; {
     description = "Linux command-line client for ProtonVPN";
     homepage = "https://github.com/protonvpn/protonvpn-cli-ng";
-    maintainers = [ maintainers.jtcoolen ];
+    maintainers = [ maintainers.jtcoolen maintainers.jefflabonte ];
```
Comment (bhipple) [inline pkgs/applications/networking/protonvpn-cli-ng/default.nix]:
```suggestion
    maintainers = with maintainers; [ jtcoolen jefflabonte ];
```
Slightly nicer.

**Review by bhipple — CHANGES_REQUESTED** [review]

File: pkgs/applications/networking/protonvpn-cli-ng/default.nix (hunk 2 above)
Comment (JeffLabonte) [reply pkgs/applications/networking/protonvpn-cli-ng/default.nix]: Making the change.

File: pkgs/applications/networking/protonvpn-cli-ng/default.nix (hunk 1 above)
Comment (JeffLabonte) [reply pkgs/applications/networking/protonvpn-cli-ng/default.nix]: great catch!

**Comment by JeffLabonte** [comment]
I have found that we were missing one dependency with this package: `iptables` . 

On a new installation, I wasn't able to connect to one of the VPNs, until I installed `iptables`. Making the changes @bhipple!

**Comment by bhipple** [comment]
LGTM, can you squash the two commits, then I'll merge?

**Review by bhipple — APPROVED** [review]

**Comment by JeffLabonte** [comment]
@bhipple Done! I will backport the commit to the 20.03 and 19.09! :D

Thank you

**Comment by bhipple** [comment]
```
[2 built, 6 copied (2.4 MiB), 0.7 MiB DL]
https://github.com/NixOS/nixpkgs/pull/81735
1 package built:
protonvpn-cli-ng
```

**Pull request merged**